apps/feedback/mix.exs

```diff
@@ -21,7 +21,7 @@ defmodule Feedback.Mixfile do
   #
   # Type "mix help compile.app" for more information
   def application do
-    [applications: [:logger, :mailgun, :timex, :briefly]]
+    [applications: [:logger, :mailgun, :timex, :briefly, :ex_aws, :ex_aws_ses, :mail]]
```

Comment:
It seems odd that we're manually specifying an `applications` key — normally this list is automatically inferred from the dependencies, and if you need a built-in Elixir application that's not started by default (like `logger`), you put it in `extra_applications`.

Reply:
perhaps, but given the hard deadline let's look into this later, once we've confirmed successful transition to SES.